support_bundle/get_support_bundle.sh

```diff
@@ -152,9 +152,11 @@ main() {
         if [[ ${VERSION_CHECK} == 1 ]]; then
             # This api endpoint is found in 6.x and above
             API_URL=$(kubectl ${KUBE_OPTS} get cm sysdigcloud-collector-config -ojsonpath='{.data.collector-config\.conf}' | awk 'p&&$0~/"/{gsub("\"","");print} /{/{p=0} /sso/{p=1}' | grep serverName | awk '{print $3}')
+            BACKEND_VERSION=6
         else
             # This api endpoint is found in 5.x and below
             API_URL=$(kubectl ${KUBE_OPTS} get cm sysdigcloud-config -o yaml | grep -i api.url: | head -1 | awk '{print $2}')
+            BACKEND_VERSION=5
```

Comment:
so technically this is v3, v4 or v5.  just have to be mindful of this for future versions as v7 will use the same format as v6, but there might be different API endpoints exposed,  so you need to be able to validate that.  so as of right now, this will work, but future versions you won't be able to differentiate.

Reply:
@dthomson-sysdig thanks, and I agree we will need to handle this differently if we need to differentiate between v6 and v7. If this works for now, can we merge and then re-factor it in the future?

Reply:
what if we do something like this:

```kubectl get deployment sysdigcloud-api -n sysdigcloud -ojsonpath='{.spec.template.spec.containers[0].image}' | awk 'match($0, /[0-9]\.[0-9]\.[0-9](\.[0-9]+)?/) {print substr($0, RSTART, RLENGTH)}'```

```Output:  6.2.1.230601185822```

Reply:
Used the version check from your analysis script. Works great!

Reply:
This is fixed by using the backend version check from the bundle analysis script.